src/pages/Resources/Resources.js

```diff
@@ -58,6 +67,18 @@ function Resources() {
           </Typography>
         )}
         <br />
+        <Pagination
```

Comment:
Should we also hide the pagination UI when there is a `searchValue`? 

That way we won't see something like this: 
![image](https://user-images.githubusercontent.com/4512699/96686564-89238b80-1333-11eb-9a61-d82a550b3ee0.png)

That "1" is clickable, leading to this UI which doesn't make sense: 
![image](https://user-images.githubusercontent.com/4512699/96686612-96d91100-1333-11eb-8330-41a4e90f9734.png)